skactiveml/classifier/_parzen_window_classifier.py

```diff
@@ -208,3 +226,9 @@ def predict_freq(self, X):
             for i in range(np.size(X, 0)):
                 F[i, :] = K[i, indices[i]] @ self.V_[indices[i], :]
         return F
+
+    def _calculate_mean_gamma(N, variance, delta=(np.sqrt(2) * 1e-6)):
+        numerator = 2 * N * np.sum(variance)
+        denominator = (N - 1) * np.log((N - 1) / delta ** 2)
+        gamma = 0.5 * denominator / numerator
```

Comment:
Check case when variance is zero and names of `numerator` and `denominator`.

Reply:
We switched the numerator and denominator name around.
Gamma was set to infinite when variance is 0. We added further handling to default to 1/n_features instead.